Building frontend assets fails

<!--
Please use this template while reporting a bug and provide as much info as possible.
Questions should be posted to https://community.grafana.com
Use query inspector to troubleshoot issues: https://community.grafana.com/t/using-grafanas-query-inspector-to-troubleshoot-issues/2630
-->

**What happened**: 
Running "yarn start" throws ts syntax error. Every other command on the building from source page, http://docs.grafana.org/project/building_from_source/, works as expected. Error:

 yarn start
yarn run v1.13.0
$ ts-node --project ./scripts/cli/tsconfig.json ./scripts/cli/index.ts core:start --watchTheme

C:\xyz\go\src\github.com\grafana\grafana\node_modules\ts-node\src\index.ts:226
    return new TSError(diagnosticText, diagnosticCodes)
           ^
TSError: ⨯ Unable to compile TypeScript:
scripts/cli/utils/useSpinner.ts:7:21 - error TS2350: Only a void function can be called with the 'new' keyword.

7     const spinner = new ora(spinnerLabel);
                      ~~~~~~~~~~~~~~~~~~~~~

    at createTSError (C:\xyz\go\src\github.com\grafana\grafana\node_modules\ts-node\src\index.ts:226:12)
    at getOutput (C:\xyz\go\src\github.com\grafana\grafana\node_modules\ts-node\src\index.ts:335:40)
    at Object.compile (C:\xyz\go\src\github.com\grafana\grafana\node_modules\ts-node\src\index.ts:368:11)
    at Module.m._compile (C:\xyz\go\src\github.com\grafana\grafana\node_modules\ts-node\src\index.ts:414:43)
    at Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Object.require.extensions.(anonymous function) [as .ts] (C:\xyz\go\src\github.com\grafana\grafana\node_modules\ts-node\src\index.ts:417:12)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**What you expected to happen**:
Frontend assets to be built.

**How to reproduce it (as minimally and precisely as possible)**:
Follow the steps on the build page until you get to running "yarn start".

**Anything else we need to know?**:
All other commands on the build page instructions work, and I can start the Grafana backend, it just has no assets to serve. It seems like some kind of syntax error in the script that is run by "yarn start".

**Environment**:
- Grafana version: The version downloaded from that page today.
- Data source type & version:
- OS Grafana is installed on: Windows 10 Enterprise.
- User OS & Browser: Windows 10 Enterprise.
- Grafana plugins: None
- Others:

NPM version: 6.4.1
Node version: 10.15.3
Go version: 1.11
gcc version: 5.1.0

--- a/scripts/cli/utils/useSpinner.ts
+++ b/scripts/cli/utils/useSpinner.ts
@@ -4,7 +4,7 @@ type FnToSpin<T> = (options: T) => Promise<void>;
 
 export const useSpinner = <T>(spinnerLabel: string, fn: FnToSpin<T>, killProcess = true) => {
   return async (options: T) => {
-    const spinner = new ora(spinnerLabel);
+    const spinner = ora(spinnerLabel);
     spinner.start();
     try {
       await fn(options);

